internal/store/store_test.go

```diff
@@ -761,7 +762,8 @@ func TestPruneBlocks(t *testing.T) {
 	assert.EqualValues(t, 1500, bs.Height())
 	assert.EqualValues(t, 1500, bs.Size())
 
-	state.LastBlockTime = time.Date(2020, 1, 1, 1, 0, 0, 0, time.UTC)
+	state.LastBlockTime, err = time.Parse(time.RFC3339, "2035-01-16T13:00:00Z")
```

Comment:
This test was failing, and I realised the logic of the test needs a timestamp in the future, rather than in the past.
@jmalicevic can you confirm?

Reply:
After a discussion with @jmalicevic, we agree this change is needed because, before PBTS, the blocks' timestamps were in year 1 AD, so by 2020 any evidence had expired. With PBTS, the blocks created here are getting timestamps in the present, so 2020 is its past which made evidences to not expire.

However, together with @jmalicevic, we have improved the fix in order not to hard-code a date in the future. New commit coming up.